packages/db/tests/config.js

```diff
@@ -0,0 +1,97 @@
+const test = require('ava');
+const sinon = require('sinon');
+const proxyquire = require('proxyquire').noPreserveCache();
+
+const { getRequiredEnvVar, getEnvConnectionConfig } = require('../dist/config');
+
+const dbConnectionConfig = {
+  username: 'postgres',
+  password: 'password',
+  database: 'postgres',
+  host: 'localhost',
+};
+
+const secretsManagerStub = sinon.stub().returns({
+  getSecretValue: (_value) => ({
+    promise: () => Promise.resolve({
+      SecretString: JSON.stringify(dbConnectionConfig),
+    }),
+  }),
+  putSecretValue: (_value) => ({ promise: () => Promise.resolve() }),
+});
+
+const undefinedSecretsManagerStub = sinon.stub().returns({
+  getSecretValue: (_value) => ({
+    promise: () => Promise.resolve({
+      SecretString: undefined,
+    }),
+  }),
+  putSecretValue: (_value) => ({ promise: () => Promise.resolve() }),
+});
+
+const badSecretsManagerStub = sinon.stub().returns({
+  getSecretValue: (_value) => ({
+    promise: () => Promise.resolve({
+      SecretString: { test: 'value' },
+    }),
+  }),
+  putSecretValue: (_value) => ({ promise: () => Promise.resolve() }),
+});
+
+test('getRequiredEnvVar returns an environment value if defined', async (t) => {
+  const result = getRequiredEnvVar('testVar', { testVar: 'testvalue' });
+  t.is(result, 'testvalue');
+});
+
+test('getRequiredEnvVar throws error if not defined', async (t) => {
+  t.throws(() => getRequiredEnvVar('testVar', {}));
+});
+
+test('getSecretConnectionConfig returns a Knex.PgConnectionConfig object', async (t) => {
+  const { getSecretConnectionConfig } = proxyquire('../dist/config.js', {
```

Comment:
Can we update `getSecretConnectionConfig` to take a `secretsmanager` parameter? That'll make testing it a little cleaner / more future proof.